ci-operator/jobs/openshift/certman-operator/openshift-certman-operator-master-postsubmits.yaml

```diff
@@ -1,5 +1,54 @@
 postsubmits:
   openshift/certman-operator:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^master$
+    cluster: build09
+    decorate: true
+    labels:
+      ci.openshift.io/generator: prowgen
+    max_concurrency: 1
+    name: branch-ci-openshift-certman-operator-master-leading-zero-check
+    spec:
+      containers:
+      - args:
+        - --gcs-upload-secret=/secrets/gcs/service-account.json
+        - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+        - --report-credentials-file=/etc/report/credentials
+        - --target=leading-zero-check
+        command:
+        - ci-operator
+        image: ci-operator:latest
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          requests:
+            cpu: 10m
+        volumeMounts:
```

Comment:
Not having any other knowledge of how this works: are these volumes needed, or were they copied from another example?